### PR TITLE
Android: Add Slot A and B Device settings to the settings UI

### PR DESCRIPTION
**Pull request by gwicks**
Just a basic implementation for now, to make it easier to change to a raw memory card if desired (helps with games like Pokemon Colosseum which have issues with save states + GCI folder saves).

Putting it under the core settings, since that follows where it gets stored in the config file.

### CODE REVIEW

**Comment by hackbar** [comment]
I'm not sure what the other platforms show, but just seeing "Slot A Device" in the menu is a little confusing. Maybe prefix with "GameCube"?

File: Source/Android/app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -117,8 +117,8 @@
     <string name="overclock_enable_description">Higher values can make variable-framerate games run at a higher framerate, requiring a powerful device. Lower values make games run at a lower framerate, increasing emulation speed, but reducing the emulated console\'s performance.</string>
     <string name="overclock_title">Emulated CPU Clock Speed</string>
     <string name="overclock_warning">WARNING: Changing this from the default (100%) WILL break games and cause glitches. Please do not report bugs that occur with a non-default clock.</string>
-    <string name="slot_a_device">Slot A Device</string>
-    <string name="slot_b_device">Slot B Device</string>
+    <string name="slot_a_device">GC Slot A Device</string>
+    <string name="slot_b_device">GC Slot B Device</string>
```
Comment (JosJuice) [inline Source/Android/app/src/main/res/values/strings.xml]: Please write GameCube instead of GC. (Or would that make it wrap awkwardly?)

Comment (gwicks) [reply Source/Android/app/src/main/res/values/strings.xml]: Okay, I'll change it to that (suppose it isn't much longer than some other settings are string length wise, now that I look at it).

**Comment by hackbar** [comment]
Instead of adding new commits, please squash/fixup your olds ones. Also do a rebase instead of a merge with master.

Basically, the commits here should look like you just wrote a really nice PR instead of showing the actual history of the reviews. This makes for a much nicer master when browsing, and makes bisecting easier & faster.

**Comment by JonnyH** [comment]
@dolphin-emu-bot  rebuild

**Comment by JosJuice** [comment]
@dolphin-emu-bot rebuild

**Pull request merged**